### DISCUSSION TITLE
Use of the Pkl Logo in downstream projects

### DISCUSSION THREAD

**Discussion by avarei** [apple/pkl#595] [Q&A] [answered]
Hi Pkl Maintainers,

I wonder if you are cool with downstream Projects using the Pkl Image (<img src="https://raw.githubusercontent.com/apple/pkl/main/.idea/icon.svg" alt="pkl logo" width="20" height="20" />) where appropriate.

In my case i would like to use it here:
https://marketplace.upbound.io/functions/crossplane-contrib/function-pkl/v0.0.1

but I also think that other projects might benefit from it. Like for example this one:
https://github.com/marketplace/actions/setup-pkl

**Comment by holzensp** [comment, marked as answer]
The logo is published under the same license (Apache 2.0) as the rest of Pkl. Your use-case looks entirely appropriate and license compliant, so feel free.
